README.md

```diff
@@ -202,6 +202,7 @@ See also [Rust — Production](https://www.rust-lang.org/production) organizatio
 * [Atomic-Server](https://github.com/atomicdata-dev/atomic-data-rust/) [[atomic-server](https://crates.io/crates/atomic_server)] - NoSQL graph database with realtime updates, dynamic indexing and easy-to-use GUI for CMS purposes. [![Release](https://github.com/atomicdata-dev/atomic-data-rust/actions/workflows/docker.yml/badge.svg)](https://github.com/atomicdata-dev/atomic-data-rust/actions/workflows/docker.yml)
 * [Databend](https://github.com/datafuselabs/databend) - A Modern Real-Time Data Processing & Analytics DBMS with Cloud-Native Architecture [![Release](https://github.com/datafuselabs/databend/actions/workflows/databend-release.yml/badge.svg)](https://github.com/datafuselabs/databend/actions/workflows/databend-release.yml)
 * [DB3 Network](https://github.com/dbpunk-labs/db3) — DB3 is a community-driven blockchain layer2 decentralized database network ![GitHub Workflow Status (with event)](https://img.shields.io/github/actions/workflow/status/dbpunk-labs/db3/ci.yml?branch=main&style=flat-square)
+* [CozoDB](https://github.com/cozodb/cozo) - A transactional, relational database that uses Datalog and focuses on graph data and algorithms. Time-travel-capable, and fast! [![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/cozodb/cozo/build.yml?branch=main)](https://github.com/cozodb/cozo/actions/workflows/build.yml)
```

Comment:
✅ More than [50 stars](https://github.com/cozodb/cozo/stargazers)
![image](https://user-images.githubusercontent.com/36519478/215411242-0264dc94-7602-4e2d-906e-77f161ae7ff0.png)

🚫 Not enough downloads (at least 2000 needed, [less than 600 found](https://crates.io/crates/cozo))
![image](https://user-images.githubusercontent.com/36519478/215411608-be860467-34f9-427d-968c-7b9297b2e9ab.png)

Maybe we need to wait until accept this package...

Reply:
The stars/downloads are an _or_ and also 2k+ stars is way over the 50 mark